### ISSUE TITLE
why don't you just use the post relation with user?

### ISSUE THREAD

**Issue by mahmoudadev** [HagarMohamedGamal/LaravelLab#1]
try to access the user data via post relation method that you created in the post model to be
like this `$post->user->name`, so you don't need to write fat function in the controller

https://github.com/HagarMohamedGamal/LaravelLab/blob/3daa887b390b6d02032abc711ed6b923cf02fec0/app/Http/Controllers/PostsController.php#L16-L22

**Comment by HagarMohamedGamal** [comment]
yes i realized that and edited it but i forgot to remove this lines
